tests/unit/test_adapter_registry.py

```diff
@@ -29,11 +29,21 @@ def test_adapter_registry_unknown_model():
         load_adapter("unknown")
 
 
-def test_adapter_registry_import_error():
+def test_adapter_registry_import_error_dice():
```

Comment:
I think it totally suffices to do so for one model. It doesn't test anything about the model/adapter, only that the adapter_registry throws if dependencies cannot be found.

Reply:
I agree, I wasn't going to add this for every model but in the rebase I had it for both so just thought I'd keep them. Happy to delete if you really object but having two tests doesn't cause any harm?

Reply:
;) No harm, I just wouldn't like to see people add that for each of their adapter ;)

Reply:
Maybe choose one of the adapters as the "generic" one (in conftest) for tests that need one adapter, but don't test anything specific about it, but, for instance, the `load_adapter`?

Reply:
I'll just get rid of it to avoid confusion. If you're confused, other implementers definitely will be!